apollo-ios-pagination/Sources/ApolloPagination/GraphQLQueryPager+Convenience.swift

```diff
@@ -5,33 +5,37 @@ import Foundation
 // MARK: - GraphQLQueryPager Convenience Functions
 
 public extension GraphQLQueryPager {
-  /// Convenience function for creating a pager that has a single query and does not transform output responses.
-  static func makeQueryPager<InitialQuery: GraphQLQuery, P: PaginationInfo>(
+
+  /// Convenience initializer for creating a pager that has a single query and does not 
+  /// transform output responses.
```

Comment:
Just for my reference – do y'all use an 80 character line divider here? I've been operating I think with 100 or 120 as my divider.

Reply:
We use 100! I really need to get a style guide and a swift-format configuration set up for the whole project one of these days.